bareops-main/src/error.rs

```diff
@@ -0,0 +1,9 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum BareopsError {
+    #[error("Failed to parse input")]

```

Comment:
```suggestion
    #[error("Failed to parse input: `{0}`")]
```

That way, the Display output will also include the inner error string